### PR TITLE
[color-swatch] First pass on adding reference tables to the docs

### PR DESCRIPTION
**Pull request by DmitrySharabin**

### CODE REVIEW

File: src/color-swatch/README.md [hunk 1]
```diff
@@ -131,4 +137,35 @@ If you don’t, the `<html>` element will be used.
...
+### Attributes & Properties
+
+| Attribute | Property | Property type | Default value | Description |
+|-----------|----------|---------------|---------------|-------------|
+| `color` | `color` | `Color` &#124; `string` | - | The current color value. |
+| `value` | `value` | `string` | - | The current value of the swatch. |
+| `size` | - | `large` | - | The size of the swatch. Currently, it is used only to make a large swatch. |
+| `property` | `property` | `string` | - | CSS property to bind to. |
+| `scope` | `scope` | `string` | `:root` | CSS selector to use as the scope for the specified CSS property. |
+| `verbatim` | `verbatim` | `boolean` | `false` | ??? |
```
Comment (DmitrySharabin) [inline src/color-swatch/README.md:153]: Don't know how to describe this attribute correctly. 🤷‍♂️

Comment (LeaVerou) [reply src/color-swatch/README.md:153]: Just don't include it for now

Comment (LeaVerou) [reply src/color-swatch/README.md]: (and then remove the example)

Comment (DmitrySharabin) [reply src/color-swatch/README.md]: Done

**Review by LeaVerou — APPROVED** [review]

**Pull request merged**